lib/roo_on_rails/railties/new_relic.rb

```diff
@@ -19,8 +19,12 @@ class NewRelic < Rails::Railtie
             abort "Aborting: newrelic.yml detected in '#{path.parent.realpath}', should not exist"
           end
 
+          sync_startup = !(ENV['NEW_RELIC_SYNC_STARTUP'].to_s.downcase == 'false')
```

Comment:
I'm a fan of using this pattern for these

```
ENV.fetch('NEW_RELIC_SYNC_STARTUP', 'YES').match?(/\A(YES|TRUE|ON|1)\Z/i)
```

Reply:
To be honest, if only we could just say it's either 0s or 1s that would make our lifes much better

Reply:
I second what Pedro said for the future. 0/1 or false/true. Handling them is much easier and cleaner:

```ruby
ENV[key].to_i == 1
ENV[key].to_s.downcase == 'true'
```